README.md

```diff
@@ -180,6 +180,8 @@ code-push release-cordova MyApp-Android --privateKeyPath ~/rsa/private_key.pem
 
 The CodePush client supports differential updates, so even though you are releasing your app code on every update, your end users will only actually download the files they need. The service handles this automatically so that you can focus on creating awesome apps and we can worry about optimizing end user downloads.
 
+*NOTE: for **Ionic** apps you need to run `ionic build` before running `cordova-release` or `cordova` command in order to build web assets.*
```

Comment:
@ruslan-bikkinin, could you please clarify why do we need to run `ionic build` command before the `cordova` command in terms of CodePush usage?

Reply:
@sergey-akhalkov this a typo, it should be `release` command instead. I'll fix it right now, thanks.